src/shims/lock.c

```diff
@@ -338,6 +338,20 @@ _dispatch_ulock_wake(uint32_t *uaddr, uint32_t flags)
 	);
 }
 
+#else
+
+static int
+_dispatch_ulock_wait(uint32_t *uaddr, uint32_t val, uint32_t timeout,
```

Comment:
please do not do that, making the build pass with such a critical piece missing is not helping. please revert.